providers/openshift_redeploy_certificate.rb

```diff
@@ -25,18 +25,22 @@ def whyrun_supported?
   execute 'Backup master stuff' do
     command 'tar czvf master-backup-$(date +%s).tar.gz /etc/origin/master /var/www/html/master --ignore-failed-read && rm -rf /var/www/html/master'
     cwd '/etc/origin'
+    only_if '[ -a /etc/origin/master ]'
   end
   execute 'Backup node stuff' do
     command 'tar czvf node-backup-$(date +%s).tar.gz /etc/origin/node /var/www/html/node --ignore-failed-read --remove-files'
     cwd '/etc/origin'
+    only_if '[ -a /etc/origin/node ]'
```

Comment:
Can use `node['cookbook-openshift3']['openshift_node_config_dir']` instead of hardcoding the directory